src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  margin-right: 4px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+  background-position: center;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.star--0 :nth-child(-n + 0),
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(.//images/star-active.svg);
```

Comment:
There is an extra slash in the URL path `url(.//images/star-active.svg)`. It should be `url(./images/star-active.svg)` to correctly reference the image file.